demisto_sdk/tests/test_files/validate_integration_test_valid_types.py

```diff
@@ -310,6 +310,32 @@
     "version": -1
 }
 
+LIST = {
```

Comment:
Confusing constant name, please either replace **or** use `as` when importing, to help maintainers what this mysterious constant is.